packages/graphql-api/package.json

```diff
@@ -47,6 +47,7 @@
     "@types/yargs": "^17.0.32",
     "graphql": "^16.8.1",
     "jest": "^29.7.0",
+    "tsconfig-paths": "^4.2.0",
```

Comment:
Isn't it a `dependencies` instead of `devDependencies`?

Reply:
Yes, it is. I have fixed it.